PWGCF/MultiparticleCorrelations/Core/MuPa-Configurables.h

```diff
@@ -49,6 +49,9 @@ struct : ConfigurableGroup {
   Configurable<vector<string>> cfBookQAEventHistograms2D{"cfBookQAEventHistograms2D", {"1-Multiplicity_vs_ReferenceMultiplicity", "1-Multiplicity_vs_NContributors", "1-Multiplicity_vs_Centrality", "1-Multiplicity_vs_Vertex_z", "1-Multiplicity_vs_Occupancy", "1-ReferenceMultiplicity_vs_NContributors", "1-ReferenceMultiplicity_vs_Centrality", "1-ReferenceMultiplicity_vs_Vertex_z", "1-ReferenceMultiplicity_vs_Occupancy", "1-NContributors_vs_Centrality", "1-NContributors_vs_Vertex_z", "1-NContributors_vs_Occupancy", "1-Centrality_vs_Vertex_z", "1-Centrality_vs_Occupancy", "0-Centrality_vs_ImpactParameter", "1-Vertex_z_vs_Occupancy", "0-MultNTracksPV_vs_MultNTracksGlobal", "0-CentFT0C_vs_CentNTPV", "0-CentFT0M_vs_CentNTPV", "0-CentRun2V0M_vs_CentRun2SPDTracklets", "1-TrackOccupancyInTimeRange_vs_FT0COccupancyInTimeRange", "1-CurrentRunDuration_vs_InteractionRate"}, "book (1) or do not book (0) this QA 2D event histogram"};
   Configurable<bool> cfFillQAParticleHistograms2D{"cfFillQAParticleHistograms2D", false, "if false, all QA 2D particle histograms are not filled. if true, only the ones for which fBookQAParticleHistograms2D[...] is true, are filled"};
   Configurable<vector<string>> cfBookQAParticleHistograms2D{"cfBookQAParticleHistograms2D", {"1-Pt_vs_dcaXY"}, "book (1) or do not book (0) this QA 2D particle histogram"};
+  Configurable<bool> cfFillQAParticleEventHistograms2D{"cfFillQAParticleEventHistograms2D", false, "if false, all QA 2D particle event histograms are not filled. if true, only the ones for which fBookQAParticleEventHistograms2D[...] is true, are filled"};
+  Configurable<vector<string>> cfBookQAParticleEventHistograms2D{"cfBookQAParticleEventHistograms2D", {"1-CurrentRunDuration_vs_itsNCls", "1-CurrentRunDuration_vs_itsNClsNegEtaEbyE", "1-CurrentRunDuration_vs_itsNClsPosEtaEbyE", "1-CurrentRunDuration_vs_Eta0804EbyE", "1-CurrentRunDuration_vs_Eta0400EbyE", "1-CurrentRunDuration_vs_Eta0004EbyE", "1-CurrentRunDuration_vs_Eta0408EbyE", "1-CurrentRunDuration_vs_Pt0005EbyE", "1-CurrentRunDuration_vs_Pt0510EbyE", "1-CurrentRunDuration_vs_Pt1050EbyE"}, "book (1) or do not book (0) this QA 2D particle event histogram"};
```

Comment:
Names `vector` and `string` are declared inside the `std` namespace. Since you do not specify the `std::` prefix, you force the users of the header to have a using directive in their files.

Reply:
I am fixng that gradually in the code, as that came just recently. Btw, are you not concerned with the trivial code bloat, if we now start prepending std:: all over the place?

Reply:
It is not trivial code bloat at all, so no, I am not concerned.
If you don't use the full names in your header, it's a bug, because the header is not valid, does not compile and requires unjustified action from anyone who includes it.
If you are concerned about the prefix in the `cxx` files, it is very useful to know where entities with common names are taken from, plus it allows to avoid bugs like `abs` vs `std::abs`.

Reply:
Does this mean that we should gradually abandon the usage of 'namespace'? 

I am aware of your O2Linter development, but since it came out of blue just recently, please give us some time to adjust the code to O2Linter requirements. There are much more pressing things, I already promised Victor to align my code with O2Linter gradually, I cannot really do it in one go.

Reply:
No, it does not mean we should abandon using namespace at all. We should just use it correctly. The `std` namespace is a bit special though, hence the O2 linter warning.
Nevertheless, the missing prefix in the header is a problem for any namespace. It's like selling cars without wheels and expecting the customers to buy them themselves.

Thanks for addressing the warnings gradually. No one expects it to be done in one go of course.